### PR TITLE
APP-10074: breadcrumb

### PR DESCRIPTION
**Pull request by jean9696**

### CODE REVIEW

File: src/Breadcrumb/Breadcrumb.style.ts [hunk 1]
```diff
@@ -1,42 +1,13 @@
 import styled from 'styled-components'
 
-import { Spacing } from './Breadcrumb.interface'
+import Icon from '../Icon/Icon'
```
Comment (flaviendelangle) [inline src/Breadcrumb/Breadcrumb.style.ts]: Import from '../Icon'

File: src/BreadcrumbItem/BreadcrumbItem.interface.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { styledAs } from '../_internal/types'
+import TextProps from '../Text/Text.interface'
```
Comment (flaviendelangle) [inline src/BreadcrumbItem/BreadcrumbItem.interface.ts]: Import from '../Text'

File: src/BreadcrumbItem/BreadcrumbItem.style.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  &:last-child {
+    max-width: initial;
+    overflow: visible;
+    color: ${palette.darkBlue[900]};
+  }
+
+  &[data-interactive='true']:not(:last-child) {
+    &:hover {
+      cursor: pointer;
+      background: ${palette.darkBlue[200]};
```
Comment (flaviendelangle) [inline src/BreadcrumbItem/BreadcrumbItem.style.ts:52]: No transition on background ?

Comment (jean9696) [reply src/BreadcrumbItem/BreadcrumbItem.style.ts:52]: my bad

**Review by flaviendelangle — APPROVED** [review]

**Pull request merged**